### ISSUE TITLE
Update compilers as of 2024-02-23

### ISSUE THREAD

**Issue by yoancutillas** [Chris2011/minifierbeans#101]
Hi, thank you for your very useful NetBeans plugin!

The compilers versions used by minifierBeans are outdated now, so I went ahead and made the updates, fixed a few bugs on my side.

I would like to share it here to help those who would like to make minifierBeans work with up to date compilers.
Hopefully the author can push a new version with these changes.

1.
replace minifierbeans/custom-packages/package.json content with
```
{
    "name": "minifierbeans-custom-packages",
    "version": "1.4.2",
    "description": "CLIs to minify JS, CSS, HTML and XML.",
    "devDependencies": {
        "cssnano": "^6.0.4",
        "google-closure-compiler": "^20231112.0.0",
        "html-minifier-terser": "^7.2.0",
        "minify-xml": "^4.4.1",
        "postcss": "^8.4.35",
        "postcss-cli": "^11.0.0"
    },
    "dependencies": {
        "browserslist": "^4.23.0"
    }
}
```

2.
in minifierbeans/custom-packages/postcss replace `bin/postcss` with `index.js` (2 occurrences).

3.
in minifierbeans/custom-packages/postcss.cmd replace `bin\postcss` with `index.js`.

___
And here is how to update the packages (on Windows):
- first, install Node.js in `C:\Program Files\nodejs\` (https://nodejs.org/)

Open a command prompt as administrator and run this command first (replace your user name in the path):
- `cd C:\Users\YOUR_USER_NAME\.netbeans\minifierbeans\custom-packages`

Commands to run once:
- `npm install npm -g`
- `npm install npx -g`

Command to run regularly to update packages:
- `npm update --save`

___
Finally, here is NetBeans configuration:

- in Netbeans > Tools > Options > HTML/JS > Node.js > 
      - Node Path = `C:\Program Files\nodejs\node.exe`
      - npm Path = `C:\Program Files\nodejs\npm.cmd`

- in Netbeans > Tools > Options > HTML/JS > Minification > Javascript > 
     - Compiler Flags = `--compilation_level SIMPLE; --language_in STABLE; --language_out ECMASCRIPT_NEXT; --jscomp_off misplacedTypeAnnotation; --emit_use_strict false; --allow_dynamic_import`

**Comment by Chris2011** [comment]
Hey @yoancutillas thx for your issue and work. So the good message is that YOU can help me here to just create a PR with your changes and I can push it and release a new version with it :). So you can help to make it better and work with up to date stuff. What do you think?

**Comment by yoancutillas** [comment]
Sure, I have made a PR (https://github.com/Chris2011/minifierbeans/pull/102),
I am not used to PR so I hope it is correctly made.

Moreover, I think it would be preferable is someone else can test these changes before release.

**Comment by Chris2011** [comment]
Thx, I will do so.

**Comment by Chris2011** [comment]
Fixed in latest release #103
